content/millennium/dstu2/medications/medication-statement.md

```diff
@@ -160,12 +288,24 @@ To successfully PUT a MedicationStatement, the following headers must be provide
     x-xss-protection → 1; mode=block
 </pre>
 
-[MedicationStatement.informationSource]: http://hl7.org/fhir/DSTU2/medicationstatement-definitions.html#MedicationStatement.informationSource
-[MedicationStatement.medication]: http://hl7.org/fhir/DSTU2/medicationstatement-definitions.html#MedicationStatement.medication_x_
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+### Errors
+
+The common [errors] may be returned.
```

Comment:
The current errors don't list 412 which would be returned if no "if-match" header is provided